FreeRTOS/FreeRTOS-Partner-Supported-Demos
# Updated freeRTOS demos for Microchip PIC24 and dsPIC family of devices

Updated freeRTOS demos for Microchip PIC24 and dsPIC family of devices

Description
-----------
Updated freeRTOS demos for Microchip PIC24 and dsPIC family of devices. Microchip already has these demos listed here https://github.com/microchip-pic-avr-examples/pic24-dspic33-freertos-demo . The demos listed on freeRTOS website are using old IDE and compilers. There is no major difference in these updated demos, the functionality of the demo remains the same. In devices with less memory the number of tasks created are less compared to higher memory devices.

Test Steps
-----------
1. Install MPLABX IDE and XC16 compilers
2. Build and Flash the image. Make the hardware configurations as mentioned in the individual demos.

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
